lib/urldata.h

```diff
@@ -254,6 +254,7 @@ struct ssl_primary_config {
   char *pinned_key;
   struct curl_blob *cert_blob;
   char *curves;          /* list of curves to use */
+  char *ca_file_pem;
```

Comment:
Why call this 'file' when it isn't a file at all? It points to a buffer holding PEM formatting, right?

Reply:
replaced all ca_file_pem by ca_info_pem